src/vm-repair/azext_vm_repair/_help.py

```diff
@@ -0,0 +1,37 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+helps['vm repair'] = """
+    type: group
+    short-summary: Auto repair commands to fix faulty VMs. Recommended to use all commands with the --verbose flag.
```

Comment:
Move the recommendation to the long-summary. Also update it with something similar to the bottom, with any other additional reasons to use --verbose. Those reasons can be spelled out in the relevant commands.
```suggestion
    short-summary: Auto repair commands to fix faulty VMs. 
    long-summary: It is recommended to use `--verbose` when running vm repair commands to track command progress.
```

Reply:
After short summary, you can add some examples:
- name: create a rescue VM "MyVM"
text: >
Az vm repair --resource-group MyResourceGroup --vm-name MyVM --auto